### PR TITLE
Create A/B test

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: wagtail_ab_testing/templates/wagtail_ab_testing/add_compare.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{% block titletag %}{% trans "Create an A/B Test" %} - {{ page.title }}{% endblock %}
+
+{% block content %}
+    {% trans "Create an A/B Test" as title %}
+    {% include "wagtailadmin/shared/header.html" with title=title subtitle=page.title tabbed=1 merged=1 %}
+
+    <div class="nice-padding">
+        <p>{% trans "This will create an A/B test comparing the published version of this page and the draft with the following differences:" %}</p>
+
+        <p>TODO: Comparison here</p>
```
Comment (kaedroho) [inline wagtail_ab_testing/templates/wagtail_ab_testing/add_compare.html:12]: We're going to come back to this

File: wagtail_ab_testing/models.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        RUNNING = 'running', __('Running')
+        PAUSED = 'paused', __('Paused')
+        CANCELLED = 'cancelled', __('Cancelled')
+        COMPLETED = 'completed', __('Completed')
+
+    page = models.ForeignKey('wagtailcore.Page', on_delete=models.CASCADE, related_name='ab_tests')
+    name = models.CharField(max_length=255)
+    variant_revision = models.ForeignKey('wagtailcore.PageRevision', on_delete=models.CASCADE, related_name='+')
+    goal_type = models.CharField(max_length=255)
+    goal_page = models.ForeignKey('wagtailcore.Page', null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
```
Comment (KalobTaulien) [inline wagtail_ab_testing/models.py:22]: Can we add a TODO comment here to turn this into a PageChooser?

**Review by KalobTaulien — APPROVED** [review]
Awesome start! Im looking forward to demoing this more

File: wagtail_ab_testing/models.py (hunk 2 above)
Comment (kaedroho) [reply wagtail_ab_testing/models.py:23]: Done, thanks!

**Comment by kaedroho** [comment]
Thanks!

**Pull request merged**